src/azul/service/avro_pfb.py

```diff
@@ -81,42 +91,52 @@ class PFBConverter:
     Relations.
     """
 
-    def __init__(self, schema: JSON):
+    def __init__(self, schema: JSON, repository_plugin: RepositoryPlugin):
         self.schema = schema
+        self.repository_plugin = repository_plugin
         self._entities: MutableMapping[PFBEntity,
                                        MutableSet[PFBRelation]] = defaultdict(set)
 
     def add_doc(self, doc: JSON):
         """
         Add an Elasticsearch document to be transformed.
         """
-        contents = doc['contents']
+        doc_copy = copy_json(doc, 'contents')
```

Comment:
```suggestion
        doc = copy_json(doc, 'contents')
```

It's more expressive to overwrite the previous value because it indicates clearly that it's not needed anymore. With two variables this requires extra work by the reader to come to that conclusion. The reason to have two variables despite one being unused is if their values represent different animals. Bit this is not the case here.